python/agents/academic-research/academic_research/agent.py

```diff
@@ -37,8 +37,8 @@
     instruction=prompt.ACADEMIC_COORDINATOR_PROMPT,
     output_key="seminal_paper",
     tools=[
-        AgentTool(agent=academic_websearch_agent),
-        AgentTool(agent=academic_newresearch_agent),
+        AgentTool(agent=academic_web_search_agent),
+        AgentTool(agent=academic_new_research_agent),
     ],
```

Comment:
@gulliantonio I also had a question on why these were set up as `AgentTool` instead of sub-agents